src/js/nav.js

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Is this meant to be check this in as part of the PR? Or is it just something you're working through removing?